### ISSUE TITLE
Broken Git LFS references

### ISSUE THREAD

**Issue by lgleim** [RobotecAI/o3de-ros2-gem#322]
**Bug description**
The repository has broken Git LFS references. 

**To Reproduce**
1. Check out the repository
2. Run `$ git lfs fetch --all` and see error message 
  ```
  fetch: 3 objects found, done.
  fetch: Fetching all references...
  [8d438c48dc194710dbec4b70245b0d34a2887124fb0e2b2bd1c49c03fd99a575] Object does not exist on the server: [404] Object does not exist on the server
  [7ac9dd09bde78f389e3725ac49d61eff109857e004840bc0bc3881739df9618d] Object does not exist on the server: [404] Object does not exist on the server
  error: failed to fetch some objects from 'https://github.com/RobotecAI/o3de-ros2-gem.git/info/lfs'
  ```

One of these is `(missing) preview.png (7ac9dd09bde78f389e3725ac49d61eff109857e004840bc0bc3881739df9618d)`

**Expected behavior**
No errors

**Comment by michalpelka** [comment]
Thank you @lgleim for raising that issue!
The problem is that in past one png file was added using LFS.
We are not using LFS here.
The recommended way is to implement this [removing-sensitive-data-from-a-repository](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository)

I also found question https://stackoverflow.com/questions/69319446/git-lfs-missing-files

I was testing it on my [fork](https://github.com/michalpelka/o3de-ros2-gem), but with no luck - cloning from my fork also gives error with `git lfs fetch --all`.

I think that we should not do that now and here. In near future we are planning to move repo to o3de-extras - it can be rectified easily when PR to o3de-extras will be prepared.
